options.go

```diff
@@ -205,3 +206,11 @@ func WithCustomMatrix(customMatrix [][]byte) Option {
 		o.customMatrix = customMatrix
 	}
 }
+
+// WithLeopard will always use leopard for encoding.
+// Note that Leopard places certain restrictions on use see other documentation.
```

Comment:
```suggestion
// WithLeopardGF16 will always use leopard GF16 for encoding.
// This is not compatible with Leopard GF8 output for shards < 256.
// Note that Leopard places certain restrictions on use see other documentation.
```

Reply:
Nit: "for shards < 256" is slightly misleading because FF8 doesn't support shards < 256. Otherwise, don't you mean "<= 256"?